willow/spec/support/vcr.rb

```diff
@@ -0,0 +1,18 @@
+require 'active_support/core_ext/numeric/time'
+require 'vcr'
+require 'webmock/rspec'
+
+
+
+VCR.configure do |config|
```

Comment:
VCR will record sensitive information in the cassettes by default, so you'll probably need to change its config in the future to exclude certain GET and POST params. Like `AUTH_TOKEN` for 3rd party integrations and lots of other stuff. These exceptions to what to record in cassettes can be added as you're writing tests, it's just something to keep in mind before committing cassettes to git.